Signal/src/ViewControllers/ConversationView/ConversationViewController.m

```diff
@@ -3994,13 +3922,15 @@ - (void)reloadInteractionForViewItem:(ConversationViewItem *)viewItem
     }];
 }
 
-- (nullable ConversationViewItem *)viewItemForIndex:(NSUInteger)index
+- (nullable ConversationViewItem *)viewItemForIndex:(NSInteger)index
 {
-    if (index >= self.viewItems.count) {
+    if (index < 0 || index >= (NSInteger)self.viewItems.count) {
         OWSFail(@"%@ Invalid view item index: %zd", self.tag, index);
         return nil;
     }
-    return self.viewItems[index];
+    ConversationViewItem *_Nullable viewItem = self.viewItems[(NSUInteger)index];
```

Comment:
When would this be nil?

Reply:
You're right; this is redundant with the `OWSFail()` above. Removed.